src/main/java/eu/hadeco/crudapi/RequestHandler.java

```diff
@@ -647,6 +647,16 @@ private void streamJsonObjectResults(ResultSet rs, PrintWriter writer, TableMeta
         }
     }
 
+    /**
+     * SMALLINT SIGNED
+     * INT UNSIGNED
+     * @param type
+     * @return
+     */
+    private boolean isSignedType(String type){
+        return type.toUpperCase().endsWith("SIGNED");
```

Comment:
This may be somewhat error prone and not very clear, `.endsWith(" UNSIGNED")` makes more sense to me. I would also change the function name to `isUnsignedType`. There is no "SIGNED" suffix.

Reply:
No problem.
I was trying to cover SIGNED and UNSIGNED.
![sign-unsigned](https://user-images.githubusercontent.com/9255997/37622038-ed3a4f3a-2bc0-11e8-8ffc-2d2cec0a819d.png)